packages/elements/src/components/ui/banners/banner/Banner.tsx

```diff
@@ -84,39 +85,25 @@ export const Banner: React.FC<BannerProps> = ({
             )}
           </div>
         </Row>
-        <Row justifyContent={"space-between"} flexGrow={1}>
-          <Column justifyContent={"center"}>
-            {headerText && (
-              <>
-                <Heading variant={"h5"}>{headerText}</Heading>
-              </>
-            )}
-            {text && (
-              <>
-                {headerText && <Space />}
-                <Text>{text}</Text>
-              </>
-            )}
+        <Row justifyContent={"space-between"} flexGrow={1} gap>
+          <Column justifyContent={"center"} gap>
+            {headerText && <Heading variant={"h5"}>{headerText}</Heading>}
+            {text && <Text>{text}</Text>}
           </Column>
           {contentRight && (
-            <>
-              <Indent />
-              <Column justifyContent={"center"}>{contentRight}</Column>
-            </>
+            <Column justifyContent={"center"}>{contentRight}</Column>
```

Comment:
We need indent here.
If the text gets long enough to wrap, and there is right content, there will be no space between text and right content.